lib/libutee/include/tee_api_defines.h

```diff
@@ -85,8 +91,13 @@
 #define TEE_LOGIN_APPLICATION           0x00000004
 #define TEE_LOGIN_APPLICATION_USER      0x00000005
 #define TEE_LOGIN_APPLICATION_GROUP     0x00000006
-#define TEE_LOGIN_KERNEL                0xA0000000
+#define RSVD_GP_DEFINED_LOGIN_START1    0x00000007
```

Comment:
The RSVD_ constants aren't, they are called something like "Reserved for future GlobalPlatform defined login types" etc.

Please remove such constants, they aren't needed by any code and I hope that people that add new codes can check the spec themselves to find a suitable code.